gubernator/view_build.py

```diff
@@ -28,8 +28,11 @@
 import view_base
 
 
-def parse_junit(xml, filename):
-    """Generate failed tests as a series of (name, duration, text, filename, output) tuples."""
+def parse_junit(xml, filename, stats):
```

Comment:
This is a weird function signature, it would probably be best to refactor this into a class later.